README.md

```diff
@@ -6,21 +6,27 @@
 A `<LinearGradient>` component for react-native, as seen in
 [react-native-login](https://github.com/brentvatne/react-native-login).
 
-## Add it to your project
+## Installation
 
-### Using Yarn
+Using Yarn
 
 ```sh
 yarn add react-native-linear-gradient
 ```
 
-### Using NPM
+Using NPM
```

Comment:
Minor: Since you're updating the line here, can you also fix the capitalization here? `NPM` -> `npm`?

Reply:
@friederbluemle fixed 👌